server/src/main/java/com/ti/server/login/controller/LoginController.java

```diff
@@ -0,0 +1,22 @@
+package com.ti.server.login.controller;
+
+import com.ti.server.login.model.UserPass;
+import com.ti.server.login.service.LoginService;
+import lombok.AllArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@AllArgsConstructor
+@RequestMapping("/api/v1")
```

Comment:
Please add the @CrossOrigin like in the test endpoint. We will need this on all our controllers.

Reply:
thank you, I added @CrossOrigin